.github/workflows/push.yml

```diff
@@ -72,6 +72,7 @@ jobs:
           # job id, not the job name)
           key: "LINUX_X64_${{ matrix.debug && 'DEBUG' || 'RELEASE' }}_${{ matrix.zts && 'ZTS' || 'NTS' }}-${{hashFiles('main/php_version.h')}}"
           append-timestamp: false
+          save: ${{ github.event_name != 'pull_request' }}
```

Comment:
This should be added to all the usages of `hendrikmuhs/ccache-action` in this file, I believe.

Reply:
Good catch, thanks. I've added it after waiting for the initial build to pass to verify that my YAML programming worked fine.

Will merge once CI is green for PHP 8.2 and after making a final verification of the log output.

Reply:
Great! Also note that another instance exists on PHP 8.3+ (the benchmarking job).

Reply:
Thanks for the heads up. For completely I've also added it to the Nightly job, even though it does not run for PRs. Commit: 402f3a923f8b9464f03b3e1e737bc77e70e6f91b